src/Components/Admin/admin.js

```diff
@@ -114,7 +114,7 @@ const useStyles = makeStyles((theme) => ({
 
 export default function Admin() {
     const classes = useStyles();
-    const user = useContext(UserContext);
+    const { user } = useContext(UserContext);
```

Comment:
Just curious, are we able to access the admin's Google Account picture? The designers have put this as a feature, but I wasn't sure if that's possible

Reply:
I haven't looked into it. Will probably need to do a different ticket for this.

My initial guess is no since you can make a firebase account with a non Google account, so that won't even be an accessible field. And specifically for DCC, they are using outlook emails so they definitely won't have a google account picture.